.vscode/problem-solving.code-workspace

```diff
@@ -0,0 +1,18 @@
+{
+	"folders": [
+		{
+			"path": ".."
+		}
+	],
+	"settings": {
```

Comment:
This is great to keep the workspace clean! Lmk if you agree!

Reply:
I agree!